### DISCUSSION TITLE
Pi-DAC+ with minidexed

### DISCUSSION THREAD

**Discussion by rlynker** [probonopd/MiniDexed#213] [General]
Hi all.  I'm going to start off by saying I'm new here and if I'm stating the obvious, I apologize.  I am very thankful for this project.  It's simply amazing to have such a diverse and powerful synth as this using some very inexpensive hardware (well, that is if you already HAVE a Pi laying around).  Thank you so much to @probonopd for your efforts and documentation on this amazing project!

I wanted to make use of this device using the Pi-DAC+ and I was not able to find any discussion here talking about that exact hat.  So after doing some research on the Pi-DAC+ pinout and use (https://pinout.xyz/pinout/pi_dac#), I was not so sure this device would work for me.  However, it DID indeed work for me with no configuration or wiring change, which is wonderful.

So if you read this and have access to a Pi-DAC+, it works just fine!  Now to figure out how to make/modify a 3D printed case using that configuration....

Thanks!

**Comment by Skeyelab** [comment]
Can confirm it works.  Interested in that case too!
